src/SFA.DAS.AANHub.Application/Common/Validators/RequestedByMemberId/RequestedByMemberIdValidator.cs

```diff
@@ -9,7 +9,6 @@ public class RequestedByMemberIdValidator : AbstractValidator<IRequestedByMember
         private const string RequestedByMemberIdEmptyErrorMessage = "RequestedByMemberId is empty";
         private const string RequestedByMemberIdNotFoundMessage = "RequestedByMemberId was not found";
         public RequestedByMemberIdValidator(IMembersReadRepository membersReadRepository) => RuleFor(x => x.RequestedByMemberId)
-                .NotNull()
                 .NotEmpty()
                 .WithMessage(RequestedByMemberIdEmptyErrorMessage)
```

Comment:
If the X-RequestedByUser header is missing, _both_ the "empty" error message and the "member ID not found (in the database)" message are displayed. We can use `When` and `Otherwise` to make it display only one.

Reply:
Not sure this matters much, but yes it would be more sensible that way

Reply:
@SawNawDfE you can use Stop Cascade mode to stop validating after first failure... there are many examples of this

Reply:
Stop cascade used